### PR TITLE
Phase 5 Challenges definition

### LINKED ISSUE #238: Determine validators rewards
Currently we're giving validators 250 DSM upon validating during an entire Primer Phase. Should we increment this number?

### LINKED ISSUE #239: Phase 5 challenges definition
In preparation of Primer Phase 5, we need to define the challenges that are going to be present inside it.

### PR DESCRIPTION
**Pull request by RiccardoM**
Closes #238
Closes #239

### CODE REVIEW

File: phase-5/update-node.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ When a new version of Desmos is released, all validators need to update their no
    ```bash
    cd ~/desmos
    git fetch -a 
-   git checkout tags/v0.6.2
+   git checkout tags/v0.8.0
    make install
    ```
```
Comment (kwunyeung) [inline phase-5/update-node.md:24]: Should be `phase-5` in the "Get Reward" command.

**Comment by kwunyeung** [comment]
@terencesflam Should we continue the validator uptime rewards based on the medium post for phase 4?

https://medium.com/desmosnetwork/invest-in-desmos-invest-by-validating-e70907be87db

File: phase-5/update-node.md (hunk 1 above)
Comment (RiccardoM) [reply phase-5/update-node.md:24]: Fixed :ballot_box_with_check:

**Comment by kwunyeung** [comment]
Hm... looks like we forgot to mention creating #hashtag as a challenge.

**Comment by RiccardoM** [comment]
> Hm... looks like we forgot to mention creating #hashtag as a challenge.

You're right

File: phase-5/README.md [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
+Phase 5 is planned to start on 10 July 2020 11:00 UTC. 
 :::
 
-Thanks to the latest Morpheus testnet (`morpheus-5000`), we've implemented the possibility for users of doing two new things: 
+Thanks to the latest Morpheus testnet (`morpheus-7000`), we've implemented the possibility for users of doing three new things: 
 
-1. tagging a person inside a post with a multimedia file; 
-2. reporting a post.
+1. creating posts containing hashtags;
+2. tagging a person inside a post with a multimedia file;
```
Comment (kwunyeung) [inline phase-5/README.md:9]:
```suggestion
2. tagging a person inside a post with a media file; 
```

File: phase-5/README.md [hunk 3]
```diff
@@ -1,19 +1,19 @@
...
 
-1. tagging a person inside a post with a multimedia file; 
-2. reporting a post.
+1. creating posts containing hashtags;
+2. tagging a person inside a post with a multimedia file; 
+3. reporting a post.
 
 Inside Phase 5 you will be able to play with those features and earn some DSM by completing the following challenges:
 
 - [Tag a person inside a post (50 Desmos Tokens)](tag-person.md)
```
Comment (kwunyeung) [inline phase-5/README.md:14]: Please also link the hashtag instruction with rewards here.

Comment (RiccardoM) [reply phase-5/README.md:14]: Done :heavy_check_mark:

**Pull request merged**